venom/rpc/reflect/stubs.py

```diff
@@ -1,10 +1,62 @@
 from venom import Message
 from venom.rpc import Stub
 from venom.rpc import http
+from venom.fields import Field, Repeat, String, Map, Bool
+
+
+class TypeMessage(Message):
```

Comment:
With all of these, you should use OpenAPI specification terms. Otherwise this is impossible to follow.

A `FieldsMessage` appears to correspond to a Definitions Object and `field_types_message()` returns a definition object for a message. A `TypeMessage` is a Reference Object. A `MethodMessage` is a Paths Object and `method_message()` returns a paths object for a method. It would help enormously if the messages were named appropriately, e.g. `PathsObject`, `PathsObjectMessage` or `PathsMessage` and this was done consistently for all messages here.